app/views/docs/getting-started-for-web.phtml

```diff
@@ -27,7 +27,7 @@ $demos = $platform['demos'] ?? [];
 
 <p>To init your SDK and interact with Appwrite services, you need to add a web platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before and click the 'Add Platform' button.</p>
 
-<p>From the options, choose to add a <b>web</b> platform and add your client app hostname. By adding your hostname to your project platform, you are allowing cross-domain communication between your project and the Appwrite API.</p>
+<p>From the options, choose to add a <b>web</b> platform and add your client app hostname. By adding your hostname to your project platform, you are allowing cross-domain communication between your project and the Appwrite API. Only web apps hosted on domains specified in a web platform will be able to make requests to your Appwrite instance, preventing unwanted access from malicious actors.</p>
```

Comment:
Can we get this ready for 0.14, and also mention something about wildcard hostnames?

Reply:
Goooood point! I'll deal with that tomorrow :D